templates/internet-of-things/smart-home.html

```diff
@@ -10,213 +10,296 @@
+        <div class="p-section--shallow">
+          <p>
+            With existing support for <a href="/certified">dozens of the most popular IoT chips</a>, and board support
+            offered as a service, you can start developing on Ubuntu now, and move to your final hardware when it's ready.
+          </p>
+        </div>
+      {%- endif -%}
+
+      {%- if slot == 'list_item_title_3' -%}
+        <h3 class="p-heading--5">Have confidence in your OTAs</h3>
+      {%- endif -%}
+
+      {%- if slot == 'list_item_description_3' -%}
+        <div class="p-section--shallow">
+          <p>
+            Over-the-air updates always have an element of risk. With <a href="/core">Ubuntu Core</a> you have the assurance
+            that a failed update will never affect your device.
+          </p>
+        </div>
+      {%- endif -%}
+
+    {%- endcall -%}
   </section>
 
-  <section class="p-strip--light">
-    <div class="row">
-      <div class="col-4">
-        <h2 class="p-heading--4">Free up resources</h2>
-        <p>
-          Security update issues are a thing of the past with Canonical's support. Let your developers <a href="/engage/embedded-linux-make-or-buy">focus on the application instead of maintenance</a>.
-        </p>
-      </div>
-      <div class="col-4">
-        <h2 class="p-heading--4">Start development earlier</h2>
-        <p>
-          With existing support for <a href="/certified">dozens of the most popular IoT chips</a>, and board support offered as a service, you can start developing on Ubuntu now, and move to your final hardware when it's ready.
-        </p>
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Universal compatibility</h2>
       </div>
-      <div class="col-4">
-        <h2 class="p-heading--4">Have confidence in your OTAs</h2>
-        <p>
-          Over-the-air updates always have an element of risk. With <a href="/core">Ubuntu Core</a> you have the assurance that a failed update will never affect your device.
-        </p>
+      <div class="col">
+        <div class="p-image-container is-cover is-highlighted">
+          {{ image(url="https://assets.ubuntu.com/v1/c8af9315-universal-compatibility.png",
+                    alt="",
+                    width="1800",
+                    height="1015",
+                    hi_def=True,
+                    loading="auto|lazy",
+                    attrs={"class": "p-image-container__image"}) | safe
+          }}
+        </div>
+        <h3 class="p-heading--5">Devices that speak a common language</h3>
+        Canonical <a href="https://ubuntu.com/blog/canonical-joins-the-connectivity-standards-alliance">recently
+        announced</a> its membership in the Connectivity Standards Alliance and its upcoming support for the Matter
+        standard. <a href="#get-in-touch" class="js-invoke-modal">Sign up</a> to receive smart home and IoT news.
```

Comment:
The `js-invoke-modal` attaches `#get-in-touch` hash to users with JS enabled. We tend to set the fallback href to a static contact page for users who do not have JS enabled.

```suggestion
        standard. <a href="/internet-of-things/contact-us" class="js-invoke-modal">Sign up</a> to receive smart home and IoT news.
```